frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -40,10 +48,6 @@ body {
 
 .ThemeNight .EmbedFrameHeader,
 .ThemeNight .EmbedFrameFooter {
-  color: var(
```

Comment:
these vars can be removed because the target element should have `color: var(--mb-color-text-primary);` already.